spec/bcx/comment_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+describe Bcx::Resources::Comment, :vcr do
+  let(:client) { Bcx::Client::HTTP.new(login: 'bcx-test-user', password: 'secret') }
+
+  describe "POST /projects/9634353/todos/185137845/comments.json" do
+    it "should create a new todo comment" do
+      comment = client.projects(9634353).todos(185137845).comments.create!(content: 'New Todo comment')
+      expect(comment.created_at).not_to be_blank
+    end
+  end
+
+  describe "POST /projects/9634353/todolists/29123043/comments.json" do
+    it "should create a new todolist comment" do
+      comment = client.projects(9634353).todolists(29123043).comments.create!(content: 'New TodoList comment')
```

Comment:
Line is too long. [110/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.